examples/camel-example-webhook/src/main/java/org/apache/camel/example/webhook/Application.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.example.webhook;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.stereotype.Component;
+
+//CHECKSTYLE:OFF
+@SpringBootApplication
+public class Application {
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+
+    @Component
+    public class Routes extends RouteBuilder {
+        @Override
+        public void configure() {
+            restConfiguration().contextPath("/camel");
```

Comment:
Is the rest configuration needed, as /camel is default

Reply:
Yes, `/camel` is the default when using servlet in spring-boot, but the rest configuration is not aware of that. The webhook component needs to know it because it has to reconstruct the external URL of the service and I though that the contextPath of the restConfiguration was the right place.